plugins/gz-packer.sh

```diff
@@ -1,19 +1,16 @@
 #!/usr/bin/env bash
 
 target=$1
-target_source=/
-# target_source=
 
 tar \
-    --exclude="${target_source}"dev/* \
-    --exclude="${target_source}"run/* \
-    --exclude="${target_source}"proc/* \
-    --exclude="${target_source}"sys/* \
-    --exclude="${target_source}"tmp/* \
-    --exclude="${target_source}"${0} \
-    --exclude="${target_source}${target}.tar.gz" \
+    --exclude={/data,/apex,/vendor,/system,/sdcard} \
+    --exclude={/dev/*,/proc/*,/sys/*,/tmp/*,/mnt/*,/media/*,/lost+found/*} \
```

Comment:
Now that we added support to termux its better exclude dev, proc, sys folders. tar may try reading file in those folders if we use patters like `/dev/*` which will cause tar to exit with failures.

> PS: fixing this myself , also adding checks for these folders in udroid.sh